documentation/docs/release-notes/0.30.0.md

```diff
@@ -0,0 +1,69 @@
+# Release Notes - Version 0.30.0
+
+🚀 Welcome to `hummingbot` version 0.30.0!
+
+
+## 💵 New command: Balance Limit
+
+As described in feature request [#1443](https://github.com/CoinAlpha/hummingbot/issues/1443), users can now apply a limit to the total balance to allocate how much the bot can access in an exchange.
+
+This command is useful when running multiple bots sharing the same assets in an account e.g. setting a 50% USDT limit to each bot running BTC-USDT and ETH-USDT pair.
+
+Run the command `balance limit [exchange] [asset] [amount]` to do this. More information in [balance command](/operation/commands/balance/#balance-limit-exchange-asset-amount).
+
+
+## 📜 New command: Balance Paper
+
+Adding paper trade balances can now be done easier from the Hummingbot client by running `balance paper [asset] [amount]`. Also, users can now check paper trading account balances by running `balance paper` command.
+
+For more information and sample usage, refer to [balance command](/operation/commands/balance/#balance-paper) in our documentation.
+
+
+## 📖 New command: Order Book
+
+Users can now see the market's order book by running the [order_book](/operation/commands/order_book) command from the Hummingbot CLI while a strategy is running. 
+
+By default, this command shows the top 5 bid/ask prices with order volume. It can be used with optional arguments like `--lines` to specify the number of lines to show, `--markets` and `--exchange` if running on cross-exchange and arbitrage strategy.
+
+
+## 📝 Continuously saving trades in CSV file
+
+In the previous versions, users can only see the list of trades in the current session from the `history` command i.e. restarting Hummingbot will start a fresh list of trades in the new session.
+
+Starting this release, all trades are now continuously saved in a CSV file that can be accessed in the `data` folder.
+
+
+## 🗄 Improved trades list management in database
+
+We changed how Hummingbot manages trades stored in the local database. This allows users to run multiple bots from the same parent directory without mixing up the list of trades in the history output.
+
+This will also allow running multiple windows of Hummingbot via binary without errors when running the `history` command.
```

Comment:
I think it's worth adding a warning here to users e.g. I've run into issues with too many bots hammering the same sqlite db too often causing crashes/errors

Reply:
Thanks, will add a note. What errors did you run into and how many bots did you try running on the same sqlite db? Please create a GitHub issue or we will try to reproduce this behavior and we'll create one.

Reply:
@dennisocana sorry just seen this but this issue: covers it https://github.com/CoinAlpha/hummingbot/issues/2201